tests/.data/README.rst

```diff
@@ -0,0 +1,2 @@
+Resolwe doesn't create `data` and `upload` directories automatically so
```

Comment:
We should probably add to Resolwe this 2 dirs and write test for a basic:file field on the process input.